lxd/db/cluster/schema.go

```diff
@@ -20,6 +20,11 @@ CREATE TABLE config (
     value TEXT,
     UNIQUE (key)
 );
+CREATE TABLE failure_domains (
```

Comment:
Given this is tied to nodes, shouldn't it be `nodes_failure_domains`?

Reply:
Done.